modules/internal/ChapelBase.chpl

```diff
@@ -840,7 +840,7 @@ module ChapelBase {
     }
   }
 
-  pragma "unsafe" // work around problems storting non-nilable classes
+  pragma "unsafe" // work around problems storing non-nilable classes
```

Comment:
I'm 90% sure this should be "storing", not "sorting".  (Or "starting" or "snorting".)